src/index.ts

```diff
@@ -38,7 +38,10 @@ async function getProviderEndpointURLForURL(
           new RegExp(scheme.replace(/\*/g, '(.*)')).test(url),
         )
       ) {
-        return {provider, endpoint: endpoint.url}
+        return {
+          provider,
+          endpoint: endpoint.url.replaceAll(/{format}/g, 'json')
```

Comment:
This can be just a string


```suggestion
          endpoint: endpoint.url.replaceAll('{format}', 'json')
```